composer.json

```diff
@@ -31,10 +31,12 @@
         "mongodb/mongodb": "^1.15"
     },
     "require-dev": {
+        "roave/security-advisories": "dev-latest",
         "phpunit/phpunit": "^10.3",
         "orchestra/testbench": "^8.0",
         "mockery/mockery": "^1.4.4",
-        "doctrine/coding-standard": "12.0.x-dev"
+        "doctrine/coding-standard": "12.0.x-dev",
+        "nunomaduro/larastan": "^2.6"
```

Comment:
I understand that larastan's interest is in its ability to boot the container to analyze the facades. We don't have any applications to boot in this library. I'd rather use phpstan directly instead of depending on an extra abstraction layer and its potential breaking changes.

Reply:
Alright, I'll replace it with "native" PHPStan